src/_includes/components/funders.njk

```diff
@@ -2,9 +2,19 @@
 	<symbol id="canada-logo" viewBox="0 0 143.11 34.11">
 		{% include 'svg/canada.svg' %}
 	</symbol>
+	<symbol id="hewlett-logo" viewBox="0 0 143.11 34.11">
+		{% include 'svg/hewlett-foundation.svg' %}
+	</symbol>
 </svg>
 
 <div class="funders">
+	<p>Funded by a grant from The William and Flora Hewlett Foundation.</p>
+  <div>
+    <a href="https://hewlett.org/">
+      <svg><use xlink:href="#hewlett-logo" /></svg>
+      <span class="screen-reader-only">Funded by a grant from The William and Flora Hewlett Foundation.</span>
```

Comment:
"The William and Flora Hewlett Foundation" should be a link pointing to [https://hewlett.org/](https://hewlett.org/).